tests/integration_test.rs

```diff
@@ -11,13 +11,13 @@ fn prefix_exists(map: &HashMap<PathBuf, String>, prefix: &str) -> bool {
 #[test]
 fn test_parse_simple() {
     let expected_files = [
-        "0_0_0/aot_forward_graph",
-        "0_0_0/dynamo_output_graph",
+        "-_0_0_0/aot_forward_graph",
+        "-_0_0_0/dynamo_output_graph",
```

Comment:
This seems not so nice. Just omit the `-_` delimiter when there is no compiled autograd. There isn't ambiguity because the first 0 can only be a digit anyway.